### PR TITLE
feat : updated rented/leased in assets module

### PR DESCRIPTION
**Pull request by saurkarankita**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/assets/add-assets/add-assets.component.ts [hunk 1]
```diff
@@ -612,10 +662,54 @@ export class AddAssetsComponent implements OnInit {
...
+          this.assetsData.assetDetails.ownAmt =
+            result.assetDetails.ownAmt;
+        }
+         if (result.assetDetails.ownerShip === "rented") {
+          this.assetsData.assetDetails.ownCurr =
+            result.assetDetails.ownCurr;
+        }
+        if (result.assetDetails.ownerShip === "rented") {
+          this.assetsData.assetDetails.ownDate =
+            result.assetDetails.ownDate;
```
Comment (FH-sandeepkaur1) [inline src/app/pages/fleet/assets/add-assets/add-assets.component.ts:678]: Please refractor this code

File: src/app/pages/fleet/assets/add-assets/add-assets.component.ts [hunk 2]
```diff
@@ -772,7 +866,20 @@ export class AddAssetsComponent implements OnInit {
...
+        ownCname: this.assetsData.assetDetails.ownCname,
+        ownAmt: this.assetsData.assetDetails.ownAmt,
+        ownCurr: this.assetsData.assetDetails.ownCurr,
+        ownDate: this.assetsData.assetDetails.ownDate,
+        ownRec: this.assetsData.assetDetails.ownRec,
+        ownFrq: this.assetsData.assetDetails.ownFrq,
+        
+       
+        
+
```
Comment (FH-sandeepkaur1) [inline src/app/pages/fleet/assets/add-assets/add-assets.component.ts:879]: Please remove extra spaces

File: src/app/pages/fleet/assets/asset-detail/asset-detail.component.html [hunk 3]
```diff
@@ -1,1216 +1,1071 @@
...
+                                                        </td>
+                                                        <td class="text-capitalize">
+                                                            {{ ownerShip ? ownerShip : ("-" | titlecase) }}
+                                                        </td>
+                                                    </tr>
+                                                    <tr *ngIf="ownerShip === 'rented'">
+                                                        <td class="back-color-gray font-weight-bold">
+                                                            Rented Company Name
+                                                        </td>
+                                                        <td class="text-capitalize">
```
Comment (FH-sandeepkaur1) [inline src/app/pages/fleet/assets/asset-detail/asset-detail.component.html:169]: Please refractor this code. You can combine both the conditions by using an OR operator for leased and rented.

**Review by FH-sandeepkaur1 — CHANGES_REQUESTED** [review]

File: src/app/pages/fleet/assets/add-assets/add-assets.component.ts (hunk 2 above)
Comment (saurkarankita) [reply src/app/pages/fleet/assets/add-assets/add-assets.component.ts:879]: ok

**Comment by saurkarankita** [comment]
Conflicts and conversation resolved in new PR here is 
https://github.com/Fleet-Hawks-Inc/fh-cloud-app/pull/2089
review and merge

**Comment by saurkarankita** [comment]
issues solved in another PR.

**Pull request closed without merging**